officemanager/department/views/views.py

```diff
@@ -1,3 +1,33 @@
 from django.shortcuts import render
```

Comment:
Please use ModelViewSet (https://www.django-rest-framework.org/api-guide/viewsets/#modelviewset) for CRUD operations instead of xxxAPIView.